app/views/checkouts/_checkouts_table.html.erb

```diff
@@ -16,23 +16,23 @@
       </thead>
       <tfoot>
       <tr>
-        <th scope="col" colspan="7"><%= render_renew_button %></th>
+        <th scope="col" colspan="6">
+          <%= submit_tag 'Renew', class: 'btn btn-primary', data: { disable_with: 'Please wait...' } %></th>
       </tr>
       </tfoot>
       <tbody>
       <% @checkouts.each do |checkout| %>
-          <tr>
+          <tr id="checkout<%= checkout.item_key %>">
             <td>
               <%= render SelectAllChildComponent, key: checkout.item_key, target_keyword: 'renewal' %>
             </td>
             <td>
               <%= render BibItemComponent, bibitem: checkout %>
             </td>
-            <td><%= checkout.renewal_count if checkout.renewal_count.positive? %></td>
-            <td><%= render_checkout_due_date checkout %></td>
-            <td><%= render_checkout_status checkout %></td>
-            <td><%= number_to_currency checkout.accrued if checkout.accrued.positive? %>
-            </td>
+            <td class="renewal_count"><%= checkout.renewal_count if checkout.renewal_count.positive? %></td>
+            <td class="due_date"><%= checkout.due_date_human %></td>
```

Comment:
so this is where you could render something like `"#{checkout.recall_status} <br>" if checkout.recalled?` or something

Reply:
Yes, if recalled, there will be more than one date listed in the due date column because it will add recall due date.

Reply:
ok so in that case I think the method in the model can just be an array of strings then, with some prepended as "Recalled" 👍 (rather than one string with some break tags as separators)